/robot_controller:
  ros__parameters:
    lite6_x_limit_lower: 0.1
    lite6_x_limit_upper: 0.305
    lite6_y_limit_lower: -0.1475
    lite6_y_limit_upper: 0.1475

    lite6_z_lift_amount: 0.01
    lite6_z_offset: 0.1475

    lite6_blend_radius: 0.0
    lite6_max_velocity_scaling_factor: 1.0
    lite6_max_acceleration_scaling_factor: 0.9
